wiki/Configuration:-Window-Rules.md

```diff
@@ -491,6 +491,8 @@ This is applied on top of the window's own opacity, so semitransparent windows w
 
 Opacity is applied to every surface of the window individually, so subsurfaces and pop-up menus will show window content behind them.
 
+Opacity can be toggle on or off for a window using the [toggle-window-opacity](./Configuration:-Key-Bindings.md#toggle-window-opacity) action.
```

Comment:
```suggestion
Opacity can be toggled on or off for a window using the [toggle-window-opacity](./Configuration:-Key-Bindings.md) action.
```
Unfortunately, our current wiki github action does not support linking to headers directly when it removes the `.md` file extension.